### PR TITLE
Add tr locales

### PR DESCRIPTION
**Pull request by mikropsoft**

### CODE REVIEW

File: app/src/main/res/values-tr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <string name="interval_year">Yıllık</string>
+    <string name="interval_lifetime">Ömür boyu</string>
+    <string name="time_ago_dhm">%1$dd %2$dh %3$dm önce</string>
+    <string name="time_ago_hm">%1$dh %2$dm önce</string>
+    <string name="time_ago_m">%1$dm önce</string>
+    <string name="time_in_dhm">%1$dd %2$dh %3$dm içinde</string>
+    <string name="time_in_hm">%1$dh %2$dm içinde</string>
+    <string name="time_in_m">%1$dm içinde</string>
+    <string name="just_now">Şu anda</string>
+    <string name="stats_average_per_hour">%1$.2f kez/saat"</string>
```
Comment (albertvaka) [inline app/src/main/res/values-tr/strings.xml:27]: Are the double quotes `"` after kez/saat intentional?

**Review by albertvaka — COMMENTED** [review]
Thanks for the translation :D I have one question before I merge though.

File: app/src/main/res/values-tr/strings.xml (hunk 1 above)
Comment (albertvaka) [reply app/src/main/res/values-tr/strings.xml:27]: Same for other lines below.

Comment (mikropsoft) [reply app/src/main/res/values-tr/strings.xml:27]: I don't know. en strings file looks like this

Comment (albertvaka) [reply app/src/main/res/values-tr/strings.xml:27]: Oops, I think that's a mistake :) Thanks for helping me catch it and thanks for the translation :D

**Pull request merged**